exonum-java-binding-core/rust/ejb-app/src/main.rs

```diff
@@ -1,12 +1,67 @@
+extern crate exonum_btc_anchoring;
 extern crate exonum_configuration;
 extern crate java_bindings;
+extern crate serde;
+#[macro_use]
+extern crate serde_derive;
 
+use exonum_btc_anchoring::ServiceFactory as BtcAnchoringServiceFactory;
 use exonum_configuration::ServiceFactory as ConfigurationServiceFactory;
-use java_bindings::exonum::helpers::fabric;
+use java_bindings::exonum::helpers::config::ConfigFile;
+use java_bindings::exonum::helpers::fabric::{self, ServiceFactory};
+
+use std::collections::{HashMap, HashSet};
+
+const PATH_TO_SERVICES_TO_ENABLE: &str = "ejb_app_services.toml";
+const CONFIGURATION_SERVICE: &str = "configuration";
+const BTC_ANCHORING_SERVICE: &str = "btc-anchoring";
+const EJB_SERVICE: &str = "ejb-service";
+
+#[derive(Serialize, Deserialize)]
+struct ServicesToEnable {
+    services: HashSet<String>,
+}
+
+fn service_factories() -> HashMap<String, Box<ServiceFactory + 'static>> {
```

Comment:
Isn't `Box<ServiceFactory>` `'static` by default?

Reply:
Should be, but compiler doesn't think so :upside_down_face: